ssv/spectest/tests/validator/test.go

```diff
@@ -0,0 +1,128 @@
+package validator
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"testing"
+
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	"github.com/bloxapp/ssv-spec/ssv"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+	"github.com/stretchr/testify/require"
+)
+
+type ValidatorTest struct {
+	Name                   string
+	Duties                 []*types.Duty
+	Messages               []*types.SSVMessage
+	OutputMessages         []*types.SSVMessage
+	BeaconBroadcastedRoots []string
```

Comment:
I noticed you don't really use this field for the tests...
Not much harm that it is here I suppose

your choice whether to remove it or not

Reply:
Which field?

Reply:
BeaconBroadcastedRoots

Reply:
it is just that in the tests itself I noticed it was always empty
but arguably that's also a check

Reply:
Oh, I see. Will take a look. Thx!